CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ General improvement for the Helm Chart:
 - add missing namespace label
 - make object name consistent
 
+- In cli, change `push` to `write` and `fetch` to `read`
```

Comment:
You can just put this bullet point on line 13

Reply:
yes !